playbooks/amc.yml

```diff
@@ -59,22 +59,53 @@
             ignored-mount-points: "^/(sys|proc|dev|var|run)($|/)"
             ignored-fs-types: "^(sys|proc|auto|shm|tmp|ns)fs$"
 
+# first, do everything we can safely do in parallel
+- name: app server common setup
+  hosts: edx
+  become: True
+  become_method: sudo
+  gather_facts: True
+  vars:
+    appsembler_roles: "../../appsembler-roles"
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - role: sudo
+    - role: aws
+    - role: "{{ appsembler_roles }}/scriabin"
+    - role: "{{ appsembler_roles }}/google-fluentd"
+    - role: "{{ appsembler_roles }}/node_exporter"
+      node_exporter_enabled_collectors:
+        - systemd
+        - supervisord
+        - textfile:
+            directory: "{{ node_exporter_textfile_dir }}"
+        - filesystem:
+            ignored-mount-points: "^/(sys|proc|dev|var|run)($|/)"
+            ignored-fs-types: "^(sys|proc|auto|shm|tmp|ns)fs$"
+
+# rolling deploy to one at a time
 - name: Configure app server instance(s)
   hosts: edx
   become: True
   become_method: sudo
   gather_facts: True
+  serial: 1
   vars:
     appsembler_roles: "../../appsembler-roles"
     migrate_db: "yes"
     openid_workaround: True
     EDXAPP_LMS_NGINX_PORT: '80'
     # Set to false if deployed behind another proxy/load balancer.
     NGINX_SET_X_FORWARDED_HEADERS: True
+  pre_tasks:
+    - name: remove from load balancer
+      service:
+        name: nginx
+        state: stopped
+      ignore_errors: yes
```

Comment:
```suggestion
      ignore_errors: yes  # Ignore the errors on this because it could potentially be the first 
                                     # deploy to a new server, in which case nginx would not yet be installed.
```